server/plugin/telemetry.go

```diff
@@ -39,7 +39,7 @@ func (p *Plugin) SendDailyTelemetry() {
 		"connected_user_count":          connectedUserCount,
 		"is_oauth_configured":           config.IsOAuthConfigured(),
 		"is_sass":                       config.IsSASS(),
-		"is_organization_locked":        config.GitHubOrg != "",
+		"is_organization_locked":        config.GitHubOrgs,
```

Comment:
The values of `is_organization_locked` is still a `bool`. Should we use `p.isOrganizationLocked()` instead?